### ISSUE TITLE
Markdown Toolbar is not responsive

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1831]
The markdown toolbar in different parts of the app is not responsive. On smaller screens, the toolbar elements are not visible.
This occurs on:

- [x] Profile page
- [x] Code
- [x] Kanban (edit cards)
- [x]  Forms (description and submit message)
- [x] Markdown slides

**Comment by ghost** [comment]
Tested on an iPhone SE (2020) running the latest iOS version, 18.5. Everything works great! 🙌🏻
